schema/url/computes/differentialAbundance.raml

```diff
@@ -0,0 +1,20 @@
+#%RAML 1.0 Library
+
+types:
+
+  DifferentialAbundancePluginRequest:
+    type: ComputeRequestBase
+    properties:
+      config: DifferentialAbundanceComputeConfig
+
+  DifferentialAbundanceComputeConfig:
+    properties:
+      collectionVariable: VariableSpec
+      comparisonVariable: VariableSpec
```

Comment:
These three are clearly related to one another. maybe a new raml type? 

ComparatorSpec:
  properties:
    variable:
    groupA:
    groupB: